mypy.ini

```diff
@@ -0,0 +1,8 @@
+[mypy]
```

Comment:
What do you think about moving this config to pyproject.toml? It'd mean one less file on a top level. See https://mypy.readthedocs.io/en/stable/config_file.html#using-a-pyproject-toml

Reply:
Good call! https://github.com/zytedata/zyte-common-items/pull/11/commits/6690ed8cbc3091f7888c70dc0b4281c32a30a556